scalyr_agent/builtin_monitors/syslog_monitor.py

```diff
@@ -41,13 +41,22 @@
 from scalyr_agent.monitor_utils.auto_flushing_rotating_file import (
     AutoFlushingRotatingFile,
 )
-from scalyr_agent.util import StoppableThread
-from scalyr_agent.json_lib import JsonObject
-from scalyr_agent.builtin_monitors.docker_monitor import (
-    get_attributes_and_config_from_labels,
+from scalyr_agent.util import (
+    StoppableThread,
+    get_parser_from_config,
 )
-from scalyr_agent.builtin_monitors.docker_monitor import get_parser_from_config
-from scalyr_agent.builtin_monitors.docker_monitor import DockerOptions
+from scalyr_agent.json_lib import JsonObject
+from scalyr_agent.scalyr_monitor import BadMonitorConfiguration
+
+docker_module_available = True
+
+try:
+    from scalyr_agent.builtin_monitors.docker_monitor import (
+        get_attributes_and_config_from_labels,
+    )
+    from scalyr_agent.builtin_monitors.docker_monitor import DockerOptions
+except ImportError:
+    docker_module_available = False
```

Comment:
Add a comment saying this is not expected to happen when using docker since we add in the docker module when we build the container image.  Sorry not exact text is not good but you can clean it up.